styleguide/source/assets/js/init.js

```diff
@@ -25,12 +25,4 @@
 		}
 	};
 
-	Drupal.behaviors.jumpMenu = {
-		attach: function (context, settings) {
-			$('.js-dropdown-select').on('change', function () {
-				window.location = $(this).find(':selected').data('url');
-			});
-		}
-	};
-
```

Comment:
Let's ask the team if this is used and remove / or add a conditional statement to see if there is a URL first.